### PR TITLE
Fix: API references

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

Sets the correct API provider.

### CODE REVIEW

File: openapi/anpr-dashboard.yaml [hunk 1]
```diff
@@ -65,7 +65,8 @@ info:
   # Chi posso contattare per informazioni sul servizio e sul suo stato?
   contact:
     email: roberto@teamdigitale.governo.it
```
Comment (sebbalex) [inline openapi/anpr-dashboard.yaml]: we should have a technical mail group for all these kind of contact.

Comment (ioggstream) [reply openapi/anpr-dashboard.yaml]: do it please, or set another email :)

Comment (sebbalex) [reply openapi/anpr-dashboard.yaml]:
```suggestion
    email: tech@teamdigitale.governo.it
```

**Review by sebbalex — APPROVED** [review]
:+1:

**Pull request merged**